tuned-main.conf

```diff
@@ -49,3 +49,27 @@ log_file_max_size = 1MB
 # It can be used to force tuning for specific architecture.
 # If commented, "/proc/cpuinfo" will be read to fill its content.
 # cpuinfo_string = Intel
+
+# Enable TuneD listening on dbus
+# enable_dbus = 1
+
+# Enable TuneD listening on unix domain socket
+# enable_unix_socket = 1
+
+# Path to socket for TuneD to listen
+# Existing files on given path will be removed
+# unix_socket_path = /run/tuned/tuned.sock
+
+# Paths to sockets for TuneD to send signals to separated by , or ;
+# unix_socket_signal_paths =
+
+# Default unix socket ownership
+# Can be set as id or name, -1 or non-existing name leaves unchanged
+# unix_socket_ownership = -1 -1
+
+# Permissions for listening sockets
+# unix_socket_permissions = 0o600
+
+# Size of connections backlog for listen function on socket
+# Higher value allows to process requests from more clients
+# connections_backlog = 16
```

Comment:
After [reading on](https://arthurchiao.art/blog/the-mysterious-container-somaxconn/) and playing with this I suggest to bump this to 1024.  Even 128 is just way too low given how infrequently new connections are processed by `TuneD`.  I'd also advise to add a note about the fact the actual value is capped at `/proc/sys/net/core/somaxconn`.  See the manual page for `listen`.